### ISSUE TITLE
Markdown plugin doesn't render nested blockquotes

### ISSUE THREAD

**Issue by Aldarone** [shaarli/Shaarli#488]
Blockquotes rendering is a bit odd with the markdown plugin.

The following text is rendered differently in Shaarli and in the demo page of Parsedown
```
## Blockquote

This is a standard paragraph. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.

> "**This is a blockquote**. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl."
>
> > "**This is a nested blockquote**. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl."
```

Parsedown : 
```
<h2>Blockquote</h2>
<p>This is a standard paragraph. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</p>
<blockquote>
<p>&quot;<strong>This is a blockquote</strong>. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl.&quot;</p>
<blockquote>
<p>&quot;<strong>This is a nested blockquote</strong>. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl.&quot;</p>
</blockquote>
</blockquote>
```

Shaarli : 
```
<h2>Blockquote</h2>
<p>This is a standard paragraph. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</p>
<blockquote>
<p>"<strong>This is a blockquote</strong>. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl."<br>
&gt;<br>
&gt; "<strong>This is a nested blockquote</strong>. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl."</p>
</blockquote>
```

**Comment by alexisju** [comment]
Probably related to https://github.com/shaarli/Shaarli/pull/470

**Comment by ArthurHoaro** [comment]
More likely to #466

**Comment by ArthurHoaro** [comment]
Should be fixed by #491
